dotcom-rendering/scripts/webpack/webpack.config.ts

```diff
@@ -63,9 +74,10 @@ const commonConfigs = ({ platform }) => ({
 		DEV &&
 			new WebpackMessages({
 				name: platform,
-				logger: (message) => {
+				logger: (message: string) => {
 					// distinguish between initial and subsequent (re)builds in console output
-					if (builds < module.exports.length * 2) {
+					// eslint-disable-next-line @typescript-eslint/no-use-before-define
+					if (builds < configs.length * 2) {
```

Comment:
Is it `configs` that's being used before definition here?

Reply:
yes, indeed, and it itself relies on `commonConfigs`, so you can’t satisfy both rules here.